drainer/util.go

```diff
@@ -273,20 +274,37 @@ type schemaInfo struct {
 	id   int64
 }
 
+// copy from dm:loader
 func getStmtFromFile(file string) (string, error) {
-	content, err := ioutil.ReadFile(file)
+	data := make([]byte, 0, 1024*1024)
```

Comment:
use a `strings.Builder` here

`data = append(data, []byte(realLine)...)` can be replaced by `data.WriteString(realLine)`.

`data[len(data)-1] == ';'` can be replaced by `strings.HasSuffix(realLine, ";")`

`query := string(data)` can be replaced by `query := data.String()`

`data = data[0:0]` can be replaced by `data.Reset()`